kfs-core/src/main/java/edu/arizona/kfs/fp/document/DisbursementVoucherDocument.java

```diff
@@ -44,6 +44,9 @@
  * Document class override to ensure that the bank code is synchronized with the
  * payment method code.
  */
+// org.kuali.rice.kns.document.authorization.TransactionalDocumentPresentationController is deprecated
+// getReportable1099TransactionsFlag() and processAfterRetrieve() use unchecked Lists
+@SuppressWarnings({ "deprecation", "unchecked" })

```

Comment:
Oh don't get me wrong, I absolutely love the idea of making comments to *why* each suppression is being used, it's actually a practice I try follow too. The issue is, I can't find an '@Deprecated' tag in the  class specified in the comments, or any its ancestors:
https://github.com/ua-eas/ksd-kc5.2.1-rice/blob/rice-2.1-ua-development/kns/src/main/java/org/kuali/rice/kns/document/authorization/TransactionalDocumentPresentationController.java

This makes me nervous, since it feels like one of us are pointed at the wrong Rice branch, perhaps. Either that, or Eclipse/IntelliJ are not representing compile warnings appropriately, and since I can't find a tag for this suppression, I'm leaning towards thinking it's an Eclipse thing.

Can you verify in your dev env why Eclipse would be throwing the warning without a deprecation tag being present? If it's an idiosyncrasy then it is what it is; I'm more or less interested in ensuring we don't have an env mismatch or similar.

(And apologies for the delay, I was out yesterday, and had to iron out last of my Rice repo + IDE config this morning.)